src/mscorlib/src/System/Collections/Concurrent/ConcurrentQueue.cs

```diff
@@ -1109,12 +1109,23 @@ internal struct Slot
         }
     }
 
+    /// <summary>A placeholder class for common padding constants and eventually routines.</summary>
+    internal static class PaddedHelper
```

Comment:
@jkotas @janvorli ping.  I can also just make the pre-existing cache size definition conditional on ARM64.  Then refer to it where it is.  Then rename and relocation can be a separate task.